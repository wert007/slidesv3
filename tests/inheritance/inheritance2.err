thread 'main' panicked at 'FlaggedWord {
    value: "#3b2",
    comment: "Return address",
}', slides\src\evaluator\memory.rs:74:9
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/c5c7d2b37780dac1092e75f12ab97dd56c30861d/library\std\src\panicking.rs:579
   1: core::panicking::panic_fmt
             at /rustc/c5c7d2b37780dac1092e75f12ab97dd56c30861d/library\core\src\panicking.rs:64
   2: slides::evaluator::memory::FlaggedWord::unwrap_value
             at C:\dev\rust\slidesv3\slides\src\evaluator\memory.rs:74
   3: slides::evaluator::evaluate
             at C:\dev\rust\slidesv3\slides\src\evaluator.rs:132
   4: slides::Project::evaluate
             at C:\dev\rust\slidesv3\slides\src\lib.rs:52
   5: reaktor::compile
             at C:\dev\rust\slidesv3\reaktor\src\main.rs:61
   6: reaktor::main
             at C:\dev\rust\slidesv3\reaktor\src\main.rs:36
   7: core::ops::function::FnOnce::call_once<enum2$<core::result::Result<tuple$<>,std::io::error::Error> > (*)(),tuple$<> >
             at /rustc/c5c7d2b37780dac1092e75f12ab97dd56c30861d\library\core\src\ops\function.rs:250
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
