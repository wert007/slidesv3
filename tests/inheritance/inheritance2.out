<p>Hello World!</p>
<p>test haha!</p>
<b>Hello World!</b>
0: 0
1: 0
2: 0
3: 0
4: 0
5: 0
= -1
